### PR TITLE
add button to frontend so admin can clear old logs

### LINKED ISSUE #14: There is a "Purge Job Log" button on the job log page that invokes that API call.

### PR DESCRIPTION
**Pull request by michaelhu88**
closes #14 

Screenshot 2024-11-26 at 2 14 40 PM

https://6734fd8a489da54c4e3feeff-fuahiguaxo.chromatic.com/?path=/story/pages-admin-adminjobspage--default

https://courses-michaelhu88.dokku-04.cs.ucsb.edu/

### CODE REVIEW

**Review by raysujf — CHANGES_REQUESTED** [review]
In AdminJobsPage, you just add a button for purge but didn't implement it. Then your entire job page is disabled in dokku.
Screen Shot 2024-11-26 at 5 52 15 PM

Also, the localhost's storybook can't be opened by others. Please use the storybook link implemented on github.

Sorry, that should have included another PR question. But please fix it as well.

**Review by raysujf — APPROVED** [review]
LGTM

File: frontend/src/main/pages/Admin/AdminJobsPage.js [hunk 1]
```diff
@@ -156,6 +156,12 @@ const AdminJobsPage = () => {
       <h2 className="p-3">Job Status</h2>
 
       <JobsTable jobs={jobs} />
+
+      <div className="mt-3">
+        <button className="btn btn-danger" data-testid="purgeJobLogButton">
+          {"Purge Job Log"}
+        </button>
+      </div>
```
Comment (pconrad) [inline frontend/src/main/pages/Admin/AdminJobsPage.js:164]: I'm confused.. does this button actually do anything?   A button typically has some kind of action associated with it.

This seems to be putting a button on the page that doesn't do anything at all.

Instead, it should be tied to a backend post call or delete call that actually purges the job log. 

Adding a button that doesn't do anything isn't particularly helpful.

**Review by pconrad — CHANGES_REQUESTED** [review]
I'm confused—what does this button do?

**Comment by pconrad** [comment]
Replaced by #51

**Pull request closed without merging**